services/src/users/postgres_userdb.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "postgres")]
```

Comment:
put the cfg(feature = "postgres") into mod.rs before "pub mod postgres_xxxdb;"

Reply:
26511745f68be89453766afe268f7d49343fa48b